nlp_models/nlp_utils/nlp_utils.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+from nlp_models.model_nlp.model_nlp import NlpModel
+from transformers import AutoTokenizer, LEDConfig, LEDForConditionalGeneration
+import numpy as np
+
+from server_utils.logger import log_function, get_current_logger
+
+
+class NlpUtils:
+    def __init__(self):
+        self.logger = get_current_logger()
+        self.nlp_model = NlpModel()
+
+    @log_function
+    def categorize_article(self, article_content: str) -> List[str]:
+        try:
+            self.logger.debug("Categorizing article")
+            category = self.nlp_model.categorize(article_content)
+            self.logger.info(f"Categorized article to `{category}`")
+            return category
+        except Exception as e:
+            self.logger.error(f"Failed categorize article")
+            raise e
+
+    def summarize(self, content: str) -> str:
```

Comment:
Check if getting all the cluster articles content and summarize using PRIMER (multi-document summarizer)